stdlib/public/core/CompilerProtocols.swift

```diff
@@ -634,7 +634,7 @@ public protocol ExpressibleByArrayLiteral {
   /// The type of the elements of an array literal.
   associatedtype ArrayLiteralElement
   /// Creates an instance initialized with the given elements.
-  init(arrayLiteral elements: ArrayLiteralElement...)
+  init(arrayLiteral elements: __consuming ArrayLiteralElement...)
```

Comment:
I thought we had a rule that initializers always take their arguments at +1 by default?

Reply:
Ah, looks like that is the case. Thanks.